src/modules/persistent-cart/persistentCartProduct/persistentCartProduct.js

```diff
@@ -82,6 +83,19 @@ const ItemPrice = styled.h4`
   }
 `
 
+const LineItemPrice = ({ price, adjustedPrice }) => {
+  if (adjustedPrice) {
+    return (
+      <ItemPrice>
+        <span className="strikeThruPrice">{accounting.formatMoney(price)}</span>
+        <span>&nbsp;&nbsp;{accounting.formatMoney(adjustedPrice)}</span>
+      </ItemPrice>
+    )
+  } else {
```

Comment:
You can skip the `else` statement as you're returning inside the `if` bit.

Reply:
Good call!